Sources/Extensions/UIKit/UIViewExtensions.swift

```diff
@@ -555,5 +555,18 @@ public extension UIView {
 		anchorCenterYToSuperview()
 	}
 
+    /// SwifterSwift: Search all superviews until a view with this class is found.
+    ///
+    /// - Parameter name: class of the view to search.
+    public func ancestorView<T: UIView>(withClass name: T.Type) -> T? {
```

Comment:
I just put the `withClass` in here because it is used like this in other methods of SwifterSwift, e.g. `UITableView`:
```
public func register<T: UITableViewCell>(cellWithClass name: T.Type) {
```